packages/nuxt/src/head/runtime/components.ts

```diff
@@ -70,6 +70,7 @@ export const Script = defineComponent({
       default: undefined
     },
     defer: Boolean,
+    fetchpriority: String,
```

Comment:
https://wicg.github.io/priority-hints/#definitions
```
type FetchPriority = 'high' | 'low' | 'auto'
```

```suggestion
    fetchpriority: String as PropType<FetchPriority>,
```